### PR TITLE
Expose navigate constants

### PR DESCRIPTION
**Pull request by tobiasandersen**
This PR addresses #257. Just let me know if there's anything else needed here!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,14 +6,15 @@ import momentLocalizer from './localizers/moment';
...
-import { views } from './utils/constants';
+import { views, navigate } from './utils/constants';
 
 Object.assign(Calendar, {
   setLocalizer,
   globalizeLocalizer,
   momentLocalizer,
   label: viewLabel,
   views,
+  navigate,
```
Comment (jquense) [inline src/index.js]: small nit, I'd capitalize this to `Navigate` to be a bit clearer it's a namespace...i should have done that with `views` as well

Comment (tobiasandersen) [inline src/index.js]: I agree. Do you prefer having one capitalized and the other not then? I.e. do you want me to only change it to `Navigate`? Could also add a duplicate for `views` called `Views`, and deprecate the lowercase in a later version...

Comment (jquense) [inline src/index.js]: >  Could also add a duplicate for views called Views, and deprecate the lowercase in a later version...

perfect

**Comment by jquense** [comment]
thanks!

**Comment by tobiasandersen** [comment]
Thanks for the quick response!

**Pull request merged**